roles/openshift_metrics/vars/main.yaml

```diff
@@ -0,0 +1,19 @@
+hawkular_tasks:
```

Comment:
To a naive reader (e.g. me), "tasks" sounds like these would be ansible tasks.
Perhaps rename to "hawkular_permission_oc_commands"?